testing/src/uploadResultTQ.js

```diff
@@ -0,0 +1,58 @@
+var request = require('request');
```

Comment:
The file must be converted to typescript.
We do not have plain javascript in the project unless it is really required.